onnxruntime/hosting/http/routes.cc

```diff
@@ -62,7 +61,6 @@ http::status Routes::ParseUrl(http::verb method,
   }
 
   if (!found_match) {
-    std::cerr << "Path not found: [" << url << "]" << std::endl;
```

Comment:
Same here. Let's keep the log for debugging.

Reply:
It will be done with an error callback.